context/app/static/js/pages/Publication/Publication.jsx

```diff
@@ -62,8 +62,18 @@ function Publication({ publication, vignette_json }) {
         <PublicationsVisualizationSection vignette_json={vignette_json} uuid={uuid} />
       )}
       <Files files={files} uuid={uuid} hubmap_id={hubmap_id} />
-      <ContributorsTable contributors={contributors} title="Authors" />
-      {shouldDisplaySection.provenance && <ProvSection uuid={uuid} assayMetadata={publication} />}
+      <ContributorsTable
+        contributors={contributors}
+        title="Authors"
+        iconTooltipText="The authors are listed in the same sequence as the author list of the publication."
```

Comment:
Can you check if this tooltip is actually true? Is the author list in that table in the same order as the publication's author list? If it isn't, remove this tooltip.

Reply:
For this one: https://portal-prod.stage.hubmapconsortium.org/browse/publication/72cbeb8ff605fd5017cb2666cd19dfb7, the order is mostly identical, but not exactly.

Reply:
@tsliaw @lchoy The list in that publication matches the order in `contributors.tsv` in Globus as far as I can tell. We could specify that `The authors are listed in the sequence specified by the publication's uploaders` or something along those lines, but I'm not sure that information is as useful/applicable to viewers as it is to the uploaders themselves

Reply:
Let's remove this tooltip, if there's any confusion in the future about the ordering of this author list, we can add back this tooltip for clarification.

Reply:
👍 Works for me! The tooltip has been removed.